tests/driver_srf02/main.c

```diff
@@ -28,26 +36,35 @@
 #error "TEST_MODE not defined"
 #endif
 
-#include <stdio.h>
+#define SAMPLE_PERIOD       (100 * 1000U)
 
-#include "xtimer.h"
-#include "srf02.h"
-#include "periph/i2c.h"
+static srf02_t dev;
 
-#define SAMPLE_PERIOD       (1000 * 1000U)
+static void sample_loop(void)
+{
+    uint32_t wakeup = xtimer_now();
 
-int main(void)
+    while(1) {
+        uint16_t distance = srf02_get_distance(&dev, TEST_MODE);
+        printf("distance = %3i cm\n", distance);
+        xtimer_usleep_until(&wakeup, SAMPLE_PERIOD);
+    }
+}
+
+static int cmd_init(int argc, char **argv)
 {
     int res;
-    srf02_t dev;
-    uint32_t wakeup = xtimer_now();
 
-    puts("\nSRF02 Ultrasonic Range Sensor Test\n");
-    puts("This test will sample the sensor once per second and display the\n"
-         "result\n");
+    if (argc < 2) {
+        printf("usage: %s <addr (decimal)>\n", argv[0]);
```

Comment:
In the header file you changed the format of `SRF02_DEFAULT_ADDR` to hex but here you ask for decimal numbers. I'd like to unify that